.github/workflows/update-snapshot.yml

```diff
@@ -56,10 +57,10 @@ jobs:
       # Commit changes to snapshot files.
       - name: Commit new snapshots
         run: >
-          git diff --exit-code --quiet -- samples/StaticHtmlComposites/tests || 
-          (git add samples/StaticHtmlComposites/tests && 
-          git commit -m 'Updating js bundle automation snapshots' && 
-          git push origin HEAD:${{ github.head_ref }})
```

Comment:
@PorterNan is there a need for `HEAD:${{ github.head_ref }}`. workflow_dispatch didn't like this and I found just `git push` worked, but mainly I've noticed that if changes have gone into main and this update-snapshot runs, it pulls those new changes into your PR and they appear in the PR changelist - hoping this fixes that issue